README.md

```diff
@@ -1,6 +1,6 @@
 # ReverseDiff
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://juliadiff.org/ReverseDiff.jl/)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://juliadiff.org/ReverseDiff.jl/stable)
```

Comment:
shouldn't we remove the dead link altogether, if there is no stable docs website?

Reply:
I deleted all outdated jekyll/mkdocs stuff from the gh-pages branch and fixed a few things in the Documenter config in this PR. So hopefully it will work.

Reply:
cool!